### ISSUE TITLE
detach().numpy() for float tensor turn 3 digit float numbers to integers

### ISSUE THREAD

**Issue by galharth** [pytorch/pytorch#12632]
flloat_torch.detach().numpy()  change the 3 digit float numbers in the tensor to integers.
for example: 102.3000 changes to 102.

What should I do?

Thank you very much

**Comment by soumith** [comment]
Please give a short 5 line reproduction of what you mean.
I cant reproduce your problem.

```
In [1]: import torch

In [2]: torch.randn(10, requires_grad=True).detach().numpy()
Out[2]:
array([ 0.8744673 , -0.30610833, -0.676061  , -0.34472573, -0.41794285,
       -0.47518116,  0.08590094, -0.4775773 ,  0.13862485,  0.79755896],
      dtype=float32)
```

**Comment by galharth** [comment]
My bad, it works.
